### DISCUSSION TITLE
Time Series  the legend could be order by Legend values

### DISCUSSION THREAD

**Discussion by Linzyoo** [grafana/grafana#38599] [Dashboards, Panels & Visualizations]
The Graph(old) support ordering the legend by legend values, and it is useful. Could we have this feature on the Time Series?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
